wntr/epanet/toolkit.py

```diff
@@ -768,6 +697,173 @@ def ENgettimeparam(self, eParam):
         self._error()
         return lValue.value
 
+    def ENaddcontrol(self, iType: int, iLinkIndex: int, dSetting: float, iNodeIndex: int, dLevel: float) -> int:
+        """
+        Add a new simple control
+
+        Parameters
+        ----------
+        iType : int
+            _description_
+        iLinkIndex : int
+            _description_
+        dSetting : float
+            _description_
+        iNodeIndex : int
+            Set to 0 for time of day or timer
+        dLevel : float
+            _description_
```

Comment:
update descriptions and add a test for add/get/set controls using the toolkit.

Reply:
Will do